java/src/Ice/src/main/java/com/zeroc/Ice/ConnectionI.java

```diff
@@ -492,189 +490,6 @@ public synchronized void asyncRequestCanceled(OutgoingAsyncBase outAsync, LocalE
     }
   }
 
-  private void sendResponse(OutgoingResponse response, boolean isTwoWay, byte compress) {
```

Comment:
I reworked/merged this code and moved it after dispatchAll. The logic didn't change.